projects/scannet_offline_eval/demo/proto_robot/robodata_server.py

```diff
@@ -1,11 +1,18 @@
+import sys
+
+sys.path.append("../../..")
```

Comment:
why do we have this?
I would rather move this into `src/home_robot/home_robot/utils/rpc` and then we dont need to do any path stuff

@alexsax  any thoughts

Reply:
Seems reasonable to me -- should we have a private fork of home-robot? I'm a little concerned with having all of our current development in a pubic repo

Reply:
I'm honestly not concerned by doing development in public, the LLM stuff is not in this repo